packages/communication-react/breaking-change-check/CompareChanges.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+export type CheckBreakingChanges<OLD, NEW extends OLD> = NEW;
+
+export type ExcludeList =
```

Comment:
These are the list we intentionally make breaking changes on, we will still check removed property and compatible property names/types
But it will skip the check for new properties in the interface (and we don't consider them as breaking change)

Reply:
Add something to this effect as a doc comment to this file/export